### ISSUE TITLE
Rename component

### ISSUE THREAD

**Issue by bnnm** [stuerp/foo_input_vgmstream#3]
vgmstream is actively mantained, and always has an official foo_input_vgmstream here: https://vgmstream.org/

Seems this project is a wrapper for the foobar repository: https://www.foobar2000.org/components/view/foo_input_vgmstream

It's confusing for users (to say the least) that a side project reuses the name of an official active project, so please rename this component.

We could add the official foo_input_vgmstream back to foobar's component repository once this is renamed, since we weren't aware it was available. kode54 didn't want to update it before, but he isn't associated with the project anymore.

Thanks.

**Comment by stuerp** [comment]
This version started from the kode54 code because there was no 64-bit version available for foobar2000 2.0. Does the project support this version yet?

**Comment by bnnm** [comment]
Yes, 64 bit support for was added recently and works in 2.0 64/32-bit or 1.6 32-bit.

**Comment by stuerp** [comment]
OK. I'll pull the component as soon as I have time.

**Comment by bnnm** [comment]
Thanks, appreciated. I'll look into adding it later.
